kubernetes/base/be-refactor/generic-consumer/deployment.yaml

```diff
@@ -52,4 +36,11 @@ spec:
               value: "0.0.0.0"
```

Comment:
`HOST` and `PORT` are no more needed (they should have been removed from applications)